content/02.introduction.md

```diff
@@ -24,69 +24,62 @@ Further, we aim to amplify the true signal out of the vast noise produced by tho
+The immune response therefore describes the complex interplay between pathogens infecting hosts and hosts responding with a multifaceted immune defense; a pathogen's virulence depends on its success in evading the immune system or otherwise manipulating the host.
+Innovation in therapeutics often seeks to supplement or bolster the immune response in order to enhance the body's ability to eliminate a pathogen.
+However, disease symptoms can be caused either by a pathogen or by the immune response to the pathogen, and a host immune system's efforts to fight an infection can also sometimes cause significant damage to the host [@doi:978-0815332183; @doi:9780071283663].
+Therefore, understanding how the host responds to SARS-CoV-2 and when this response becomes dangerous is critical to the treatment of COVID-19 and to the identification and/or development of possible therapeutics.
+
+##### SARS-CoV-2 and the Immune Response
+
+The first line of defense for the immune system is physical barriers such as the skin and mucus, but viruses such as SARS-CoV that infect the epithelium are able to bypass these defenses [@doi:10.1038/mi.2015.127].
+Once a pathogen enters circulation, the innate immune response uses several mechanisms to try to eliminate it.
+Several types of leukocytes (white blood cells) are professional phagocytic cells that can detect and engulf a particle, enveloping it in a phagosome.
+The conditions within the phagosome are then modulated to destroy the pathogen through fusion of the phagosome with a lysosome or granule to form a phagolysosome.
+This allows the cell to break the pathogen into its component peptides. 
+Monocytes, which include both dendritic cells and macrophages, are types of phagocytic white blood cells that are also professional antigen-presenting cells, which meaning that they can then present the resulting viral peptides on the cell surface via Major Histocompatability Complex II (MHC-II) proteins.
+However, research in related viruses suggests that SARS-CoV-2 may possess some mechanisms that facilitate evasion of the antibody-mediated response.
+_In vitro_ analysis of nidoviruses including SARS-CoV suggests that these viruses can induce the development of double-membrane vesicles, preventing the dsRNA signature from being detected by the host immune system [@doi:10.1128/jvi.02501-05].
+This protective envelope can therefore insulate these coronaviruses from the innate immune response's detection mechanism [@doi:10.1016/j.jpha.2020.03.001].
+Presentation of the antigen is important because this step allows it to be detected by B-lymphocytes, also called B-cells, which are a type of white blood cell with antibodies bound to the membrane.
+Individual B-cells have unique membrane-bound antibodies.
+A B-cell is able to bind to an antigen only if its antibody is a match.
```

Comment:
@rando2 Yes, that's what I meant BUT(!) then I saw that actually even though my suggestion is technically correct, even some biology textbooks use the term antibody a bit more loosely in some ways probably because it sort of makes more sense but they still usually follow that "antibody" implies soluble and then sometimes say something like "membrane-bound antibody" to specify that they are referring to the receptor. (& immunoglobulin can refer to either soluble or receptor) To cover both ways especially b/c what you originally wrote uses more common terminology, you could also write:
"Individual B-cells have unique membrane-bound immunoglobulins (i.e. membrane-bound antibodies).
A B-cell is able to bind an antigen only if its immunoglobulin is a match."
Very sorry for the over-correction!! aak!